### ISSUE TITLE
Separate into lib and CLI

### ISSUE THREAD

**Issue by noffle** [noffle/ipget#14]
Break this repo apart into `ipget-cli` and `ipget-lib`. Projects like [gx](https://github.com/whyrusleeping/gx) can take advantage of ipget's functionality, while users can continue to benefit via the CLI app.

**Comment by whyrusleeping** [comment]
Yeah, i've been wanting to have a way for it to work without a daemon running locally. would `ipget-lib` support creating 'sessions'? So i can do multiple fetches on a given ephemeral daemon?

**Comment by noffle** [comment]
I haven't thought too deeply on the api yet, but that use case makes
good sense given how expensive bringing up a node is. :+1:

**Comment by faddat** [comment]
Closed by #91
